tools/ci/retry.ps1

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env pwsh
+
+if ($args.Length -lt 1) {
+    Write-Host "Usage: $($MyInvocation.InvocationName) <command> [arguments]"
+    Write-Host "The script will execute the provided command and retry in case of failures"
+    Exit 1
+}
+
+$NB_RETRIES = 5
+$command = [string]::Join(' ', $args)
```

Comment:
Is there a better way to pass arguments? If some arguments have escapes / spaces the command is broken.
I don't know enough powershell but we can use `"$@"` in posix shells to use provided arguments but keep the escapes